packages/uploadthing/src/internal/handler.ts

```diff
@@ -474,16 +475,13 @@ export const buildRequestHandler = <
 
         return {
           cleanup: promise,
-          body: parsedResponse.map((x) => ({
-            ...x,
-            pollingUrl: generateUploadThingURL(`/api/serverCallback`),
-          })),
+          body: parsedResponse satisfies UTEvents["upload"]["out"],
```

Comment:
we handle this mapping on infra, no need for mapping
![CleanShot 2024-03-03 at 19 53 08@2x](https://github.com/pingdotgg/uploadthing/assets/51714798/0649b885-a286-41b5-8aa0-fa3db92b9e59)